src/sonic-yang-mgmt/setup.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+from setuptools.command.build_py import build_py
+from os import system
+from sys import exit
+
+# important reuirements parameters
+build_requirements = ['../../target/debs/stretch/libyang_1.0.73_amd64.deb',
```

Comment:
Please declare path to target.

Reply:
I will try to use, the $LIBYANG with $DEST, let`s see if this works.

Reply:
Will do the same in setup.py of src.sonic-yang-models.

Reply:
Done, using os environ vars now